index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <a class="log">My Logo</a>
+    <nav>
+      <img src="img/Union.png" alt="bar">
+    </nav>
+  </header>
+  <main>
+    <section class="background-img row">
+        <div class="main">
+          <h1 class="main-description">
+            <br />I'm Tushar
+            <br />Glad to see
+            <br />you
+          </h1>
+        </div>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or website. Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="text">let's connect</div>
+        <ul class="button-bott">
+          <li><a href=""><img class="twitter" src="img/twiter.png" alt="twiter_logo"></a></li>
+          <li><a href=""><img class="linkedIn" src="img/linkedin.png" alt="linkedIn_logo"></a></li>
+          <li><a href=""><img class="m" src="img/m.png" alt="m_logo"></a></li>
+          <li><a href=""><img class="github" src="img/github.png" alt="github_logo"></a></li>
+          <li><a href=""><img class="ok" src="img/ok.png" alt="ok_logo"></a></li>
+        </ul>
+    </section>
+    <section class="project">
+    <div class="card">
+      <img class="pic" src="img/Snapshoot Portfolio.png" alt="First-img" />
+      <h2 class="card-name">Tonic</h2>
+      <ul class="can-bak">
+        <li class="cann">canopy</li>
+        <li class="bak">Back End Dev</li>
+        <li class="bak">2015</li>
+      </ul>
+      <p class="tex-sec">A daily selection of privately personalized read;no account or signs-up required</p>
+      <ul class="work-list">
+      <li class="work-name">html</li>
+        <li class="work-name">css</li>
+        <li class="work-name">javascript</li>
+      </ul>
+      <button class="sec-button">see project</button>
```

Comment:
- [ ] [OPTIONAL] Please add the type attribute to all the buttons as this is used to define the purpose of the button.  [Read more about this here](https://dev.to/clairecodes/why-its-important-to-give-your-html-button-a-type-58k9).
E.g 
```
<button class="sec-button" type="button" >see project</button>
```